XF/Mvc/Dispatcher.php

```diff
@@ -25,7 +25,9 @@ public function dispatchClass(
                     break;
             }
 
-            $action = sprintf('%s/%s', $method, $action);
+            if ($params && !$params->offsetExists('_isApiReroute')) {
```

Comment:
Cái này sợ bị sai logic không nhỉ? Nếu mà xài `!$param || !isset($params['_isApiRoute'])` thì an toàn hơn?

Reply:
`!$params` thì rơi vô case mình ko biết. Nếu bỏ qua thì sợ có vấn đề thôi a.

Reply:
Trong PR này đang bỏ qua case không có $params đấy chứ.